code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -808,7 +808,7 @@
 	icon_state = "bombersyndie"
 	item_state = "bombersyndie"
 	ignore_suitadjust = FALSE
-	allowed = list(/obj/item/flashlight, /obj/item/tank/internals/emergency_oxygen, /obj/item/toy, /obj/item/storage/fancy/cigarettes, /obj/item/lighter)
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals/emergency_oxygen, /obj/item/toy, /obj/item/storage/fancy/cigarettes, /obj/item/lighter, /obj/item/gun, /obj/item/melee/classic_baton/telescopic/contractor, /obj/item/kitchen/knife/combat)
```

Comment:
`/obj/item/gun` do you want any guns to be storable in the jacket? I only ask because this would allow stuff like the BSG, RPGs, and long-arms to be storable in the jacket.

Reply:
Well generally yes, you think its bad? They are super rare anyway